### PR TITLE
Fix error: 'name 'rq' is not defined'

### LINKED ISSUE #108: [bug] Missing variable on the project.
**Describe the bug**
The functions get_address and get_coordinates uses the variable **rq** not created. 
I think is missing the lib requests on the project and the import requests as rq on top of the file.

**To Reproduce**
```python
from flask_googlemaps import get_address, get_coordinates
API_KEY = 'YOUR API KEY'

#Reverse Geocoding: getting detailed address from coordinates of a location
print(get_address(API_KEY,22.4761596,88.4149326))
#output: {'zip': '700150', 'country': 'India', 'state': 'West Bengal', 'city': 'Kolkata', 'locality': 'Kolkata', 'road': 'Techno City', 'formatted_address': 'Sirin Rd, Mauza Ranabhutia, Techno City, Kolkata, West Bengal 700150, India'}


#Geocoding: getting coordinates from address text
print(get_coordinates(API_KEY,'Netaji Subhash Engineering College Kolkata'))
#output: {'lat': 22.4761596, 'lng': 88.4149326}
```

### PR DESCRIPTION
**Pull request by Hona**
Closes #108 

Simply missing the `requests` module

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -113,3 +113,4 @@ wcwidth==0.1.8; python_version >= "3.4" \
 werkzeug==1.0.0 \
     --hash=sha256:6dc65cf9091cf750012f56f2cad759fa9e879f511b5ff8685e456b4e3bf90d16 \
     --hash=sha256:169ba8a33788476292d04186ab33b01d6add475033dfc07215e6d219cc077096
+requests==2.23.0; python_version >= "3.4"
```
Comment (rochacbruno) [inline requirements.txt]: This project is using Poetry the way to add a new dep is

1 -add to poetry
```
poetry add requests
```
or include manually on pyproject.toml dependencies list.

then run `poetry export -f requirements.txt -o requirements.txt` to export it to the requirements.txt with the proper hashes.

**Comment by TheRealHona** [comment]
When poetry exported the requirements it removed a lot of requirements, so I only added the new ones for `request`.

**Comment by TheRealHona** [comment]
Fixed up now

**Comment by Riverfount** [comment]
Hi @TheRealHona,

Please solve the conflicts. After that we can merge this PR!

**Comment by TheRealHona** [comment]
Fixed merge conflicts and updated the broken dependencies. Do you want me to rebase the bad commits away?

**Pull request merged**